### DISCUSSION TITLE
Thanks for such an incredible Firmware

### DISCUSSION THREAD

**Discussion by JEBurroughs** [mriscoc/Ender3V2S1#338] [General]
I am not a compiler, I just can't seem to get a grip on it. Doesn't matter though because everything I have wanted or was advanced enough to even have a clue about is in the August build firmware. I am simply blown away at all the functionality. 

https://www.youtube.com/watch?v=DnbpnXjENGk

**Comment by mriscoc** [comment]
> I am not a compiler, I just can't seem to get a grip on it. Doesn't matter though because everything I have wanted or was advanced enough to even have a clue about is in the August build firmware. I am simply blown away at all the functionality.
> 
> https://www.youtube.com/watch?v=DnbpnXjENGk

You are welcome.
